readme.md

```diff
@@ -162,13 +233,24 @@ Your Rails app has the following `application.html.erb`. Nothing shows up in you
 </html>
 ```
 
-> Your answer...
+```
+ANSWER:
+
+<!DOCTYPE html tag is not closed and the title tag is not closed as well.  It should be:
+
+  <!DOCTYPE html>
+  <title>
+```
```

Comment:
Correct, but there's one other subtle one, which is the `yield` ERB tag needs an equal sign!